web-app/preprocess/classification.py

```diff
@@ -0,0 +1,74 @@
+import sys
+import nltk
+import string
+from nltk.corpus import stopwords
+from nltk.stem import WordNetLemmatizer
+import os
+import pickle
+import numpy as np
+import fasttext.util
+from scipy.spatial.distance import cosine
+# Remove "other" from list of stopwords in order to classify crime types such as "other crime", "etc"
+
+def simplifyText(crimeType):
+    crimeTypeLower = crimeType.lower()
+    crimeTypeNoHyphen = crimeTypeLower.replace('-', ' ')
+    crimeTypeNoPunct = crimeTypeNoHyphen.translate(str.maketrans("", "", string.punctuation))
+
+    crimeTypeWords = nltk.word_tokenize(crimeTypeNoPunct)
+    crimeTypeWordsNoStopwords = [word for word in crimeTypeWords if word not in stopwords.words('english')]
+    lemmatizer = WordNetLemmatizer()
+    lemmatizedCrimeTypeWords = [lemmatizer.lemmatize(word) for word in crimeTypeWordsNoStopwords]
+    
+    return lemmatizedCrimeTypeWords
+
```

Comment:
Questionable nit: my IDE usually requires 2 blank lines after a function def, not sure if that's Google standard though